JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -65,22 +70,25 @@
     ///////////////////////////////////////////////////////////////////////////////////////
      -->
 
+
     
 
 
     <!-- http://ontology.eil.utoronto.ca/icontact.owl#hasAddress -->
 
+
     <owl:ObjectProperty rdf:about="http://ontology.eil.utoronto.ca/icontact.owl#hasAddress">
         <rdfs:range rdf:resource="http://ontology.eil.utoronto.ca/icontact.owl#Address"/>
         <rdfs:comment>A relationship between a spatial entity and its address.</rdfs:comment>
-        <rdfs:isDefinedBy>http://ontology.eil.utoronto.ca/icontact.owl</rdfs:isDefinedBy>
-        <rdfs:label>has address</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://ontology.eil.utoronto.ca/icontact.owl"/>
```

Comment:
Do we not need a </rdfs:isDefinedBy>at the end of this line? This comment is valid for all of the instances of <rdfs:isDefinedBy>

Reply:
No George mentioned that the current TBoxGeneration class in JPS_BASE_LIB handles this correctly.